Dockerfile

```diff
@@ -20,25 +20,24 @@ RUN apt-get update && \
 
 ENV LC_ALL=C.UTF-8
 
-
 COPY pyproject.toml /pretalx
 COPY src /pretalx/src
 COPY deployment/docker/pretalx.bash /usr/local/bin/pretalx
 COPY deployment/docker/supervisord.conf /etc/supervisord.conf
 
-RUN pip3 install -U pip setuptools wheel typing && \
-    pip3 install -e /pretalx/[mysql,postgres,redis] && \
-    pip3 install pylibmc && \
-    pip3 install gunicorn
+RUN pip install -U pip setuptools wheel typing && \
```

Comment:
These changes (removing suffix `3`) are unnecessary and even less safe.

Reply:
yes @hongquan fixed the changes